src/configs/messages.js

```diff
@@ -16,9 +14,6 @@ const messages = {
   PATTERN_REMOVE_MESSAGE: 'Ви впевнені, що хочете видалити цей гобелен?',
   DELETE_SIZE_MESSAGE: 'Ви впевнені, що хочете видалити цей розмір?',
   NO_SIZES_MESSAGE: 'Розміри відсутні',
-  NO_CATEGORY_MESSAGE: 'Категорії відсутні',
```

Comment:
Why these confs were removed?
Rows 19-21

Reply:
this issue is fixed successfully

Reply:
this issue is fixed successfully

Reply:
this issue is fixed successfully

Reply:
this issue is fixed successfully